openmc/data/data.py

```diff
@@ -273,6 +277,31 @@ def atomic_weight(element):
                          .format(element))
 
 
+def half_life(isotope):
+    """Return half-life of isotope in seconds or None if isotope is stable
+
+    Half-life values are from the `ENDF/B-VIII.0 decay sublibrary
+    <https://www.nndc.bnl.gov/endf-b8.0/download.html>`_.
+
+    Parameters
+    ----------
+    isotope : str
+        Name of isotope, e.g., 'Pu239'
+
+    Returns
+    -------
+    float
+        Half-life of isotope in [s]
+
+    """
+    global _HALF_LIFE
+    if not _HALF_LIFE:
+        # Load ENDF/B-VIII.0 data from JSON file
+        half_life_path = Path(__file__).with_name('half_life.json')
+        _HALF_LIFE = json.loads(half_life_path.read_text())
+
+    return _HALF_LIFE.get(isotope.title())
```

Comment:
Ok, evidently this fails for metastables because, e.g., `'Ag109_m1'.title()` evaluates to `'Ag109_M1'`, but the "m" is not capital in the JSON file. It may be better to have the JSON file store nuclide names as lowercase and then use `isotope.lower()` here (this is what we do for `atomic_mass` currently). Can you make that change (as well as adding a test for a metastable)?

Reply:
Yep I can do that, thanks for spotting that edge case.